Src/library/base/elks/support/reflected_copy_semantics_object.e

```diff
@@ -118,20 +118,16 @@ feature {NONE} -- Implementation
 
 	dereference (ptr: POINTER; offset: INTEGER): POINTER
 		external
-			"C inline use %"eif_built_in.h%""
-		alias
-			"return *(EIF_REFERENCE *) eif_obj_at($ptr, $offset);"
+			"built_in static"
 		end
 
 	object_from_address (ptr: POINTER): ANY
 		external
-			"C inline use %"eif_built_in.h%""
-		alias
-			"return (EIF_REFERENCE) $ptr;"
+			"built_in static"
```

Comment:
I thought it was eventually possible to reuse the {ISE_RUNTIME}.reference_field_at_offset ?

Reply:
I will check

Reply:
so `return (EIF_REFERENCE) $ptr;"` is equal to `return *(EIF_REFERENCE *) eif_obj_at($ptr,0));` if yes I can do  `Result :=  {ISE_RUNTIME}.raw_reference_field_at_offset (ptr, 0)`